ballerina-integrator/real-time-error-notifier/main.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/file;
+import ballerina/io;
+import ballerina/log;
+import ballerina/time;
+import ballerinax/slack;
+
+listener file:Listener directoryService = new (path = directoryToListen, recursive = false);
+
+service file:Service on directoryService {
+    remote function onModify(file:FileEvent event) {
+        do {
+            string filePath = event.name;
+            log:printInfo("File modified:" + event.name);
+            if event.name.endsWith(".log") {
+                string[] logFileLines = check io:fileReadLines(filePath);
+                if logFileLines.length() > 0 {
+                    string lastLog = logFileLines[logFileLines.length() - 1];
+                    if lastLog.startsWith("ERROR:") {
+                        log:printInfo("Error detected in file:" + filePath);
+                        string timestamp = time:utcToString(time:utcNow());
+                        string slackMessage = "Error notification\n" + timestamp + ": " + lastLog;
+                        slack:ChatPostMessageResponse slackResponse = check slackClient->/chat\.postMessage.post({channel: slackChannel, text: slackMessage});
+                        if slackResponse.ok == true {
```

Comment:
Can we do
```suggestion
                        if slackResponse.ok {
```
?

Reply:
Fixed with b900d166fbb44df14cb9c29b09d6563d5a4473f2